example/k3d/scripts/smoke-test.bash

```diff
@@ -35,15 +35,16 @@ set -euo pipefail
 # Constants
 ROOT=$(git rev-parse --show-toplevel)
 K3D_CLUSTER_NAME="agent-smoke-test"
-MUTATION_FREQUENCY="5m"
-CHAOS_FREQUENCY="30m"
```

Comment:
Is there a reason we went away from the human-readable format for the durations?

Reply:
BSD sleep doesn't support the nice format 😞